volmdlr/wires.py

```diff
@@ -4317,6 +4398,11 @@ def validate_closing_point(closing_point_index, list_closing_point_indexes,
             closing_point_index = list_closing_point_indexes[-1]
         elif math.isclose(ratio, -1, abs_tol=0.3):
             closing_point_index = list_closing_point_indexes[-1]
+        elif closing_point_index - list_closing_point_indexes[-1] > 5:
+            if list_closing_point_indexes[-1] != ratio_denom-1 and\
+                    list_closing_point_indexes[-1] + 4 <= ratio_denom-1:
+                closing_point_index = list_closing_point_indexes[-1] + 4
```

Comment:
You don't need to check inequality twice, since the first one is included in the second one. And you don't need the if here.
```suggestion
        elif closing_point_index - list_closing_point_indexes[-1] > 5 and \
            list_closing_point_indexes[-1] + 4 <= ratio_denom-1:
                closing_point_index = list_closing_point_indexes[-1] + 4
```

Reply:
Well noticed, thank you!!